test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -1047,6 +1044,7 @@ async function candidateNetworkIsNotValidated(driver) {
   await driver.fill('[data-testid="network-form-ticker-input"]', 'cTH');
   await blockExplorerURLInputEl.fill('https://block-explorer.url');
 
+  await driver.delay(500);
```

Comment:
to fix a flaky test

Reply:
We should stick to predefined numbers like `regularDelayMs`